lib/iex/lib/iex.ex

```diff
@@ -222,6 +222,16 @@ defmodule IEx do
       ...(1)> #iex:break
       ** (TokenMissingError) iex:1: incomplete expression
 
+  ## Exiting the shell
```

Comment:
Not sure, but maybe we want to copy (probably move) this information in `IEx.Helpers` to have it displayed on `h()` call.
What does everyone think?